libraries/Microsoft.Bot.Builder.Alexa/Integration/AspNet.Core/AlexaRequestHandler.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Bot.Builder.Alexa.Integration.AspNet.Core
+{
+    public class AlexaRequestHandler
```

Comment:
>AlexaRequestHandler [](start = 17, length = 19)

oh. 
I heard you had middleware to translate the alexa object types, but I was assuming BF middleware.  This approach will have to be completely different in each language.  
Is there a way to do it via the adapter?  I haven't thought deeply about this but if you can do it in the adapter then it is easier to port to multiple languages in platform agnostic way.